static/js/CalendarsPage.jsx

```diff
@@ -28,7 +28,7 @@ const CalendarsPage = ({multiPanel, initialWidth}) => {
   const weeklyListings  = makeListings(weeklyCalendars);
 
   const about = multiPanel ? null :
-    <Modules type={"AboutLearningSchedules"} />
```

Comment:
Looks like the `import` statement was missed here; you use `SidebarModules` without importing it, and didn't remove the old `Modules`.

Reply:
Great catch, thank you *so* much!